backend/usecase/article_usecase.go

```diff
@@ -12,20 +17,59 @@ type articleUsecase struct {
 }
 
 func NewArticleUsecase(articleRepository article.ArticleRepository) ArticleUsecase {
-	return &articleUsecase{articleRepository: articleRepository}
+	return &articleUsecase{
+		articleRepository: articleRepository,
+	}
 }
 
 // 記事一括取得
-func (u *articleUsecase) GetArticles() error {
-	// feeds テーブルからデータを取得
-	_, err := u.articleRepository.GetArticlesAll()
-	// feed, err := u.articleRepository.GetFeeds()
+func (u *articleUsecase) GetArticles(category string, tag string, limitParam int, offsetParam int) (dto.Response, error) {
 
+	articles, err := u.articleRepository.GetArticlesByCategoryAndTag(category, tag, int(limitParam), int(offsetParam))
 	if err != nil {
-		return err
+		initializer.Logger.Error("Failed to get articles.", "error", err)
```

Comment:
ここは return しないと後続の処理が行われてしまいそうです！
```suggestion
		initializer.Logger.Error("Failed to get articles.", "error", err)
		return err
```